src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -454,6 +451,8 @@ bool core::handle_command_line(const boost::program_options::variables_map& vm)
         }
     }
 
+    if (!mocknet_read_cli_for_mocknet_arg(vm, m_service_node))
```

Comment:
tiniest nitpick ever, perhaps, but simply calling this function doesn't necessarily (at least in this codebase) imply that "mocknet" is fundamentally required condition here.  I imagine future-me could read this and wonder what that will return, and assume (but not know for sure) that "mocknet" is false.

Reply:
On second glance, I was just overthinking.